cypress-E2E-tests/cypress/integration/device_properties_widget.cy.ts

```diff
@@ -285,88 +264,409 @@ describe("Device properties widget", function () {
+    it('TC_Device_Property_Measurement_015-1', () => {
+      addDataPointConfiguration();
+      cy.get(asset_properties_widget_elements.computedPropertySelectorConfigButton).click();
+      chooseDeviceForDataPointConfiguration();
+      cy.chooseAssetOrDevice(device, 3);
+      cy.addOrRmoveDataPoint(temperatureDataPointLabel, 'add');
+      cy.get(asset_properties_widget_elements.addDataPointsButton).click();
+      cy.get(asset_properties_widget_elements.datapointSelectionList).should(
+        'contains.text',
+        max_active_data_points_message
+      );
+      cy.get(asset_properties_widget_elements.computedPropertyConfigSaveButton).should(
+        'be.disabled'
+      );
+      cy.switchDataPointToggleButton(flowDataPointLabel);
+      cy.switchDataPointToggleButton(temperatureDataPointLabel);
+      cy.get(asset_properties_widget_elements.datapointSelectionList).should(
+        'contains.text',
+        min_active_data_points_message
+      );
+      cy.switchDataPointToggleButton(flowDataPointLabel);
+      cy.get(asset_properties_widget_elements.datapointSelectionList).should(
+        'not.contains.text',
+        min_active_data_points_message
+      );
+      cy.get(asset_properties_widget_elements.computedPropertyConfigSaveButton).should(
+        'be.enabled'
+      );
+      cy.get(asset_properties_widget_elements.datapointSelectionList).should(
+        'not.contains.text',
+        max_active_data_points_message
+      );
+      cy.get(asset_properties_widget_elements.computedPropertyConfigSaveButton).click();
+    });
+
+    // Verify the presence of '+' and '^' button icons.
+    // On click of '+',user should be able to the respective data point. Once added,user should be able to see the added data point on selected data point preview window on the left.
+    // Once added, '+' button icon must be changed to '-' button icon on click of which user should be able to remove the data point.Once removed,user should not be able to see the added data point on selected data point preview window on the left.
+    // On click of '^', user should be able to expand the fragment series template.
+    // Verify the presence of added fragment and series name,data point template dropdown and other details having label and unit information.
+    // User should be able to select multiple data points at once.Once selected add data point button should be enabled.
+    // User must be able to see multiple data fragments after selecting a device on data selector window.
+    it('TC_Device_Property_Measurement_015-2', () => {
+      cy.get(asset_properties_widget_elements.addPropertyButton).click();
+      cy.selectProperty(last_measurement_str);
+      cy.get(asset_properties_widget_elements.selectButton).click();
+      chooseDeviceForDataPointConfiguration();
+      cy.get('c8y-ui-empty-state')
+        .should('contain.text', 'No data points to display.')
+        .and('contain.text', 'No data points selected.');
```

Comment:
I believe you may encounter a lint error due to this chaining. Please check once.

Reply:
Will handle it once Cypress lint is configured.